edgelet/iotedged/docker/linux/arm32v7/base/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM arm32v7/debian:9
```

Comment:
currently edge hub and edge agent base image based on mcr.microsoft.com/dotnet/core/runtime:2.1.10-bionic-arm32v7; we don't use same image for iotedged docker image, is it because dotnet is not needed, right? There are other  linux docker images, e.g. arm32v7/ubuntu (52.4M), alpine (3.7M); why use arm32v7/debian?

Reply:
Yes, you are right. We don't need dotnet installed for iotedged images. I chose debian because 
* we use it for a public preview and it worked fine; 
* iotedged has difficulties to compile and work with musl based images; 
* we need to install additional libraries.

I'm going to make a quick research and try debian slim for a smaller image though.

Reply:
As a result of my research I switched to debian9-slim. It reduced size of base image to 48.2MB. Thanks! 
c-sdk requires libssl and libcrypto version 1.0.2 and there is no such packages for ubuntu, but debian has it. I'd like to stick with it at least so far.